rskj-core/src/test/java/co/rsk/rpc/netty/Web3HttpServerTest.java

```diff
@@ -118,6 +119,10 @@ private void smokeTest(String contentType, String host, InetAddress rpcAddress,
             String responseBody = response.body().string();
             JsonNode jsonRpcResponse = OBJECT_MAPPER.readTree(responseBody);
 
+            if (jsonRpcResponse.isEmpty()) {
+                throw JsonMappingException.from(jsonRpcResponse.traverse(), "Empty response");
```

Comment:
I wouldn't throw exceptions in a Test, I would do nothing here actually, its under our control and this should not be reached but for our mistake.

Reply:
Not necessary anymore with the latest changes